src/components/black-friday/HeroSection.tsx

```diff
@@ -4,74 +4,111 @@ import ButtonDeal from '@/components/black-friday/components/ButtonDeal';
 import Countdown from '@/components/components/Countdown';
 import Header from '@/components/shared/Header';
 import { getImage } from '@/lib/getImage';
+import Image from 'next/image';
 
 const HeroSection = ({ textContent, lang }) => {
+  const langMapping = {
```

Comment:
Why? You can use the custom hook provided by next called useRouter to get the language.